llm.py

```diff
@@ -56,10 +57,10 @@ def generate_full(prompt: str) -> str:
     def gen(
         prompt, model_generation_args, num=1, return_hiddens=False, **kwargs
     ) -> List[str]:
-        args = {**model_generation_args, **kwargs}
+        modelArgs = {**model_generation_args, **kwargs}
         num = num or 1
         model_input = tokenizer(prompt, return_tensors="pt").to("cuda")
-        input_ntokens = model_input["input_ids"].size(1)
+        input_ntokens = model_input["input_ids"].size(1) * model_input["input_ids"].size(0)
```

Comment:
why is this change not under flag?

Reply:
This is an unrelated fix in the earlier code – model_input[input_ids].size(0) is 1 usually, but doesn't have to be.

Reply:
Great!